scripts/process_schemas.py

```diff
@@ -266,7 +288,10 @@ def writeHeader(self, outputFile):
         print(file=outputFile)
         print("import avro.schema", file=outputFile)
         print(file=outputFile)
-        versionStr = self.version[1:]  # Strip off leading 'v'
+        if self.version[2] == '.':
+            versionStr = self.version[1:]  # Strip off leading 'v'
+        else:
+            versionStr = self.version
         print("version = '{0}'".format(versionStr), file=outputFile)
```

Comment:
The intent is to allow the user to set the `version = ...` line in `_protocol_definitions.py` without having to edit the file directly.  But yes, I agree the test is brittle.  I'll fix that.